### ISSUE TITLE
Apparent mismatch between the members list in the main userscript and members.json

### ISSUE THREAD

**Issue by joefarebrother** [lachlanhunt/cckufi-flair#10]
I noticed that Minecraft developer /u/Dinnerbone had a novote flair, so I decided to ctrl-f through the ccfiande list to see which position he was in, but he wasn't there. I then ctrl-f'd through members.json, and he still wasn't there, but when I searched cckufi-flair.user.js, he was. So there is a mismatch between the 2 lists.

After copying and pasting the 2 objects and writing some code to compare them, I found that there were 689 names in the main script abstain list that weren't in members.json. While these may be the people that were kicked from sukoku during the merge, or from some t15 room, there isn't any evidence of their existence anywhere else in the repository, which I feel there should be.

I've put the full list of extra names in the following pastebin: http://pastebin.com/vWijbAqa

**Comment by lachlanhunt** [comment]
Yes, these additional names came from pull #8 which included names from soKuku that dropped before the merge. I need to put all the names into a file named sokuku.csv and update members.js to incorporate that.
